presto-jdbc/pom.xml

```diff
@@ -80,8 +80,24 @@
             <artifactId>guava</artifactId>
             <exclusions>
                 <exclusion>
-                    <groupId>*</groupId>
-                    <artifactId>*</artifactId>
+                    <artifactId>checker-qual</artifactId>
```

Comment:
Why do these need to be excluded?

Also, groupId conventionally comes before artifactId. This made me look twice.

Reply:
Done: groupid before artifactid.

Only failureaccess is needed, all others are excluded.